compat/src/internal.d.ts

```diff
@@ -1,9 +1,30 @@
-import { CoreVNode } from '../../src/internal';
-export { PreactElement, ComponentFactory } from '../../src/internal'
-export { ComponentChildren, Ref, ComponentFactory, FunctionalComponent } from '../../src/index';
+{ Ref } from '../../src/index';
+{
+  Component as PreactComponent,
+  VNode as PreactVNode,
+  FunctionalComponent as PreactFunctionalComponent
+} from '../../src/internal';
```

Comment:
what happened here?

Reply:
This imports [Component type of Preact](https://github.com/developit/preact/blob/b84b873ee454ab545adc4259fe223e42feffdda4/src/internal.d.ts#L40) and gives an alias `PreactComponent`.

Reply:
but the `import` is missing

Reply:
Thank you for pointing out.
I fixed it.